src/map/script.c

```diff
@@ -12349,9 +12349,18 @@ static BUILDIN(getstatus)
  *------------------------------------------*/
 static BUILDIN(debugmes)
 {
-	const char *str;
-	str=script_getstr(st,2);
-	ShowDebug("script debug : %d %d : %s\n",st->rid,st->oid,str);
+	struct StringBuf buf;
+	StrBuf->Init(&buf);
+
+	if (!script->sprintf_helper(st, 2, &buf)) {
+		StrBuf->Destroy(&buf);
+		script_pushint(st, 0);
+		return false;
+	}
+
+	ShowDebug("script debug : %d %d : %s\n", st->rid, st->oid, StrBuf->Value(&buf));
+	StrBuf->Destroy(&buf);
+	script_pushint(st, 1);
```

Comment:
whats the `script_pushint` doing here ?
its not mention anywhere in the documentation

I guess its to return value based on getting use of sprintf_helper ?

Reply:
it's to return true/false whether or not the message is printed to the console, because `sprintf_helper` can fail

Reply:
there you go @AnnieRuru, I updated the docs